app.py

```diff
@@ -1690,8 +1737,9 @@ def send_emails_to_students(group, project, evaluation_name, from_email, path_to
                 index += 1
                 myLock = FileLock(path_to_html+'.lock')
                 with open(path_to_html, "r") as file_to_html:
-                    subprocess.call(["mail", "-s", subject, "-r", from_email, "-a", path_to_html, email])
-                    dateTimeObj = datetime.datetime.now()
+                    #subprocess.call(["mail", "-s", subject, "-r", from_email, "-a", path_to_html, email])
```

Comment:
looks like you are committing testing specific code

you can handle the need to both preserve testing specific code and push general code to github by either:
1. maintaining a separate branch (e.g., fixEmailsMyTests) on top of fixEmails. You would just keep it on your local computer to switch between quickly. Any changes to fixEmails can be added to fixEmailsMyTests by checking out fixEmailsMyTests then doing "git merge fixEmails"
2. Or, you can use git add's patch feature, the -p flag. This let's you choose only the pieces of the file that you actually want to include in the commit. you can read about it: https://johnkary.net/blog/git-add-p-the-most-powerful-git-feature-youre-not-using-yet/  or ask me at our next meeting. Bit of a learning curve but very helpful.